Doc/whatsnew/3.11.rst

```diff
@@ -676,10 +676,10 @@ enum
 * Changed :meth:`Enum.__format__() <enum.Enum.__format__>`
   (the default for :func:`format`, :meth:`str.format` and :term:`f-string`\s)
   of enums with mixed-in types (e.g. :class:`int`, :class:`str`)
-  to also include the class name in the output, not just the member's key.
+  to include the class name and member key in the output, not the member's value.
   This matches the existing behavior of :meth:`enum.Enum.__str__`,
   returning e.g. ``'AnEnum.MEMBER'`` for an enum ``AnEnum(str, Enum)``
-  instead of just ``'MEMBER'``.
+  instead of the member's value.
```

Comment:
I think it might be useful to suggest how users can keep the <3.11 behaviour (and advertise the new enum classes at the same time).

```suggestion
  instead of the member's value. Those wishing to keep the old formatting behavior may wish to modify their enums with mixed-in types to inherit from one the appropriate :class:`StrEnum` or :class:`IntEnum` class.
```
https://github.com/python/cpython/issues/100458#issuecomment-1365859762

Reply:
This sounds perfect 👍